Translation of Lisk documentation into Indonesian (#262)
LiskHQ/lisk-docs: i18n/ind/docusaurus-plugin-content-docs/current/docs-user/staking/staking.md

---

> title: Ikhtisar

Reviewer: ```suggestion
title: Gambaran Umum
```

> title: Ikhtisar

Reviewer: ```suggestion
title: Gambaran Umum
```

sidebar_position: 1
slug: /user/staking/overview
description: 'Cara untuk staking token LSK untuk menerima hadiah staking.'
keywords:
  [
    'Lisk',
    'staking Lisk',
    'staking LSK',
    'hadiah staking',
    'hadiah Lisk',
    'dapatkan LSK',
    'dapatkan Lisk',
  ]
---

# Staking Token LSK

Staking LSK diperlukan untuk berpartisipasi dalam [Lisk DAO](https://www.tally.xyz/gov/lisk), memungkinkan keterlibatan aktif dalam pengambilan keputusan proyek. Sistem staking kami memberikan hadiah kepada staker LSK atas partisipasi mereka dengan [hadiah staking](#staking-rewards) berdasarkan durasi komitmen mereka.

## Panduan Staking

> - [Cara staking, klaim, dan unstake](stake-unstake.mdx)

Reviewer: ```suggestion
- [Cara staking, klaim, dan buka staking](stake-unstake.mdx)
```

> - [Cara menghentikan posisi kunci](pause-position.mdx)

Reviewer: ```suggestion
- [Cara menghentikan posisi penguncian](pause-position.mdx)
```

> - [Cara menambah stake](../staking/increase-stake.mdx)

Reviewer: ```suggestion
- [Cara menambah staking](../staking/increase-stake.mdx)
```

- [Cara memperpanjang durasi stake](extend-duration.mdx)

> - [Cara unstake lebih awal](early-unlock.mdx)

Reviewer: ```suggestion
- [Cara membuka staking lebih awal](early-unlock.mdx)
```

> - [Cara unstake lebih awal](early-unlock.mdx)

Reviewer: ```suggestion
- [Cara membuka staking lebih awal](early-unlock.mdx)
```

> ## Posisi Kunci

Reviewer: ```suggestion
## Posisi Penguncian
```

> Setiap kali Anda [stake](stake-unstake.mdx#how-to-stake-your-lsk-tokens) sejumlah LSK untuk durasi kunci tertentu, posisi **kunci** baru akan dibuat.

Reviewer: ```suggestion
Setiap kali Anda melakukan [Staking](stake-unstake.mdx#how-to-stake-your-lsk-tokens) sejumlah LSK untuk durasi penguncian tertentu, posisi **penguncian** baru akan dibuat.
```

:::tip

> **Durasi kunci minimum** untuk staking adalah **dua minggu**.

Reviewer: ```suggestion
**Durasi penguncian minimum** untuk staking adalah **dua minggu**.
```

> **Durasi kunci maksimum** untuk staking adalah **dua tahun**.

Reviewer: ```suggestion
**Durasi penguncian maksimum** untuk staking adalah **dua tahun**.
```

:::

> Posisi kunci diwakili sebagai **NFT** menggunakan standar ERC-721, mirip dengan posisi likuiditas Uniswap v3.

Reviewer: ```suggestion
Posisi penguncian diwakili dengan **NFT** menggunakan standar ERC-721, mirip dengan posisi likuiditas Uniswap v3.
```

Ini meningkatkan komposabilitas ke dalam DeFi dan penggunaan potensial di masa depan.

## Hadiah Staking

Hadiah staking memberikan insentif bagi pengguna untuk berpartisipasi dalam tata kelola, menjadikan pengambilan keputusan lebih terdesentralisasi dan didorong oleh komunitas.  

> Secara singkat, pengguna mendapatkan [kekuatan voting](docs-user/governance/overview.mdx#voting-power) dan hadiah untuk staking token selama periode waktu tertentu.

Reviewer: ```suggestion
Secara singkat, pengguna mendapatkan [kekuatan suara](docs-user/governance/overview.mdx#voting-power) dan hadiah untuk staking token selama periode waktu tertentu.
```

Lisk menerapkan **peningkatan bobot berdasarkan waktu** pada hadiah:

> Semakin jauh akhir durasi kunci di masa depan, semakin banyak hadiah yang Anda dapatkan.

Reviewer: ```suggestion
Semakin jauh akhir durasi penguncian di masa depan, semakin banyak hadiah yang Anda dapatkan.
```

Ini juga berarti bahwa hadiah akan berkurang setiap hari.
Dengan cara ini, kami memberi penghargaan kepada pemegang token yang berkomitmen jangka panjang pada proyek Lisk sehingga kami menarik mereka sebagai pendukung jangka panjang Lisk sambil mengurangi spekulasi jangka pendek dan volatilitas.

- Pengguna dapat [mengklaim](stake-unstake.mdx#how-to-claim-staking-rewards) hadiah mereka kapan saja.   

> - Pengguna dapat memodifikasi posisi kunci mereka (menambah jumlah, memperpanjang durasi kunci) kapan saja.

Reviewer: ```suggestion
- Pengguna dapat memodifikasi posisi penguncian mereka (menambah jumlah, memperpanjang durasi penguncian) kapan saja.
```

> - Pengguna memiliki opsi untuk **menghentikan** hitungan mundur durasi kunci mereka.

Reviewer: ```suggestion
- Pengguna memiliki opsi untuk **menghentikan** hitungan mundur durasi penguncian mereka.
```

> Ini berarti, durasi kunci yang tersisa tetap tetap sampai pengguna memutuskan untuk melanjutkannya.

Reviewer: ```suggestion
Ini berarti, durasi penguncian yang tersisa tetap sama sampai pengguna memutuskan untuk melanjutkannya.
```

> Ini berarti, durasi penguncian yang tersisa tetap tetap sampai pengguna memutuskan untuk melanjutkannya.

Reviewer: ```suggestion
Ini berarti, durasi penguncian yang tersisa tetap sama sampai pengguna memutuskan untuk melanjutkannya.
```

> Perhatikan bahwa ini menghasilkan **hadiah staking yang lebih tinggi**, karena bobot tetap tetap, dibandingkan dengan bobot yang menurun setiap hari saat hitungan mundur aktif.

Reviewer: ```suggestion
Perhatikan bahwa ini menghasilkan **hadiah staking yang lebih tinggi**, karena bobot tetap sama, dibandingkan dengan bobot yang menurun setiap hari saat hitungan mundur aktif.
```

> Perhatikan bahwa ini menghasilkan **hadiah staking yang lebih tinggi**, karena bobot tetap tetap, dibandingkan dengan bobot yang menurun setiap hari saat hitungan mundur aktif.

Reviewer: ```suggestion
Perhatikan bahwa ini menghasilkan **hadiah staking yang lebih tinggi**, karena bobot tetap sama, dibandingkan dengan bobot yang menurun setiap hari saat hitungan mundur aktif.
```

### Distribusi Hadiah Staking
Sebanyak 24 juta token LSK akan digunakan selama 3 tahun untuk program hadiah staking LSK (8 juta LSK per tahun).

> Jumlah ini diberikan ke dalam [kontrak Rewards](https://blockscout.lisk.com/address/0xD35ca9577a9DADa7624a35EC10C2F55031f0Ab1f).

Reviewer: ```suggestion
Jumlah ini diberikan kepada [kontrak hadiah](https://blockscout.lisk.com/address/0xD35ca9577a9DADa7624a35EC10C2F55031f0Ab1f).
```

> **Perhitungan Hadiah Harian:** Hadiah diambil dari saldo kontrak Rewards dan dihitung setiap hari, berdasarkan jumlah yang dikunci dan durasi kunci yang tersisa.

Reviewer: ```suggestion
**Perhitungan Hadiah Harian:** Hadiah diambil dari saldo kontrak hadiah dan dihitung setiap hari, berdasarkan jumlah yang dikunci dan durasi penguncian yang tersisa.
```

Secara konkret, bobot untuk jumlah yang dikunci diberikan oleh:

```

> bobot = jumlahDikunci * (durasiKunciTersisaDalamHari + 150)

Reviewer: ```suggestion
bobot = jumlahDikunci * (durasiPenguncianTersisaDalamHari + 150)
```

```

Jumlah total hadiah staking harian, 8.000.000 / 365 LSK, kemudian dibagikan kepada semua pengguna secara proporsional dengan bobot mereka.

> Artinya, hadiah harian untuk posisi kunci dihitung sebagai:

Reviewer: ```suggestion
Artinya, hadiah harian untuk posisi penguncian dihitung sebagai:
```

```
hadiahHarian = bobot/totalBobot * TotalHadiahHarian
```

> Nilai saat ini dari `totalBobot` dapat dicek di [kontrak Rewards](https://blockscout.lisk.com/address/0xD35ca9577a9DADa7624a35EC10C2F55031f0Ab1f?tab=read_proxy).

Reviewer: ```suggestion
Nilai saat ini dari `totalBobot` dapat dicek di [kontrak hadiah](https://blockscout.lisk.com/address/0xD35ca9577a9DADa7624a35EC10C2F55031f0Ab1f?tab=read_proxy).
```

```

The total daily amount of staking rewards, 8,000,000 / 365 LSK, is then shared by all users proportional to their weight.
Jumlah total hadiah staking harian, 8.000.000 / 365 LSK, kemudian dibagikan kepada semua pengguna secara proporsional dengan bobot mereka.

That means, the daily rewards for a locking position are calculated as: 
Artinya, hadiah harian untuk posisi kunci dihitung sebagai:

```
dailyRewards = weight/totalWeight * TotalDailyRewards
hadiahHarian = bobot/totalBobot * TotalHadiahHarian
```

The current value of `totalWeight` can be checked in the [Rewards contract](https://blockscout.lisk.com/address/0xD35ca9577a9DADa7624a35EC10C2F55031f0Ab1f?tab=read_proxy).
`totalDailyRewards` can be calculated by taking the yearly available rewards and dividing it by 365:  `8,000,000 / 365 = 21917`
Nilai saat ini dari `totalBobot` dapat dicek di [kontrak Rewards](https://blockscout.lisk.com/address/0xD35ca9577a9DADa7624a35EC10C2F55031f0Ab1f?tab=read_proxy).
`TotalHadiahHarian` dapat dihitung dengan membagi hadiah tahunan yang tersedia dengan 365:  `8.000.000 / 365 = 21917`

### Example: Reward calculation for 1 vs 2 years

> For example, if someone stakes 100LSK for *1 year*, this would result in the following reward for that day, assuming `totalWeight` is `12462098705` for this example:

Reviewer: @TalhaMaliktz "100LSK" -> "100 LSK"

> Misalnya, jika seseorang melakukan staking 100LSK selama *1 tahun*, ini akan menghasilkan hadiah berikut pada hari itu, dengan asumsi `totalBobot` adalah `12462098705` untuk contoh ini:

Reviewer: ```suggestion
Misalnya, jika seseorang melakukan staking 100 LSK selama *1 tahun*, ini akan menghasilkan hadiah berikut pada hari itu, dengan asumsi `totalBobot` adalah `12462098705` untuk contoh ini:
```

> Jika 100 LSK yang di-stake habis dalam *dua tahun*, ini akan menghasilkan hadiah berikut pada hari itu:

Reviewer: ```suggestion
Jika 100 LSK yang di-staking habis dalam *dua tahun*, ini akan menghasilkan hadiah berikut pada hari itu:
```

> Setelah periode kunci dari sebuah [posisi kunci](#locking-positions) berakhir, Anda dapat membuka kembali token yang di-stake dengan cara [unstake](stake-unstake.mdx#how-to-unstake-your-lsk-tokens).

Reviewer: ```suggestion
Setelah periode penguncian dari sebuah [posisi penguncian](#locking-positions) berakhir, Anda dapat membuka kembali token yang di-staking dengan cara [buka staking](stake-unstake.mdx#how-to-unstake-your-lsk-tokens).
```

> Jika Anda perlu membuka token Anda lebih awal dari periode kunci yang ditentukan, Anda dapat melakukan unstake token Anda sebelum periode kunci berakhir dengan menggunakan opsi [early unlock](#early-unlock).

Reviewer: ```suggestion
Jika Anda perlu membuka token Anda lebih awal dari periode penguncian yang ditentukan, Anda dapat membuka staking token Anda sebelum periode penguncian berakhir dengan menggunakan opsi [buka kunci lebih awal](#early-unlock).
```

> Menggunakan unlock cepat akan melibatkan penalti yang harus dibayar.

Reviewer: ```suggestion
Membuka kunci dengan cepat akan melibatkan penalti yang harus dibayar.
```

> Setelah di-unstake, token LSK dapat digunakan seperti sebelumnya untuk tujuan lainnya.

Reviewer: ```suggestion
Setelah dibuka stakingnya, token LSK dapat digunakan seperti sebelumnya untuk tujuan lainnya.
```

> Unlock cepat memungkinkan Anda untuk melakukan unstake token LSK sebelum periode kunci mereka berakhir.

Reviewer: ```suggestion
Unlock cepat memungkinkan Anda untuk membuka staking token LSK sebelum periode penguncian mereka berakhir.
```

> Penalti = 0.5 * Jumlah yang di-stake * (Hari Tersisa / 730)

Reviewer: ```suggestion
Penalti = 0.5 * Jumlah yang di-staking * (Hari Tersisa / 730)
```

> Misalnya, jika seorang pengguna ingin melakukan unlock cepat posisi dengan 100 LSK dan durasi kunci yang tersisa 150 hari, maka penalti dihitung sebagai berikut:

Reviewer: ```suggestion
Misalnya, jika seorang pengguna ingin membuka kunci dengan cepat untuk posisi 100 LSK dan durasi penguncian yang tersisa 150 hari, maka penalti dihitung sebagai berikut:
```

The penalty is calculated using the formula:
### Unlock Cepat
Unlock cepat memungkinkan Anda untuk melakukan unstake token LSK sebelum periode kunci mereka berakhir.
Ini melibatkan pembayaran penalti dan menunggu periode unlock darurat selama 3 hari.
Penalti dihitung berdasarkan formula:

```
Penalty = 0.5 * Staked Amount * (Remaining Days / 730)
Penalti = 0.5 * Jumlah yang di-stake * (Hari Tersisa / 730)
```

Please note that the maximum value for the remaining days is 730.
Harap dicatat bahwa nilai maksimum untuk hari yang tersisa adalah 730.

For example, if a user wants to fast unlock a position with 100 LSK and a remaining locking duration of 150 days, then the penalty is calculated as follows:
Misalnya, jika seorang pengguna ingin melakukan unlock cepat posisi dengan 100 LSK dan durasi kunci yang tersisa 150 hari, maka penalti dihitung sebagai berikut:

```
Penalty = 0.5 x 100 x (147 / 730) = 10.068 LSK
Penalti = 0.5 x 100 x (147 / 730) = 10.068 LSK
```

Note that we multiply by 147 and not 150, since the user will still have to wait for 3 days, so the locking duration decreased by 147 overall. 

> 

Reviewer: @TalhaMaliktz "so the locking duration decreased by" -> "so the locking duration is decreased by"

Reviewer: @TalhaMaliktz are you sure it is "decreased by 147 overall", not "decreased by 3 overall"? 🤔

Reviewer: @fujianggara10 may need to change the Indonesian translation, based on Talha's reply later.

> Perhatikan bahwa kami mengalikan dengan 147 dan bukan 150, karena pengguna masih harus menunggu selama 3 hari, jadi durasi kunci secara keseluruhan berkurang sebanyak 147.

Reviewer: ```suggestion
Perhatikan bahwa kami mengalikan dengan 147 dan bukan 150, karena pengguna masih harus menunggu selama 3 hari, jadi durasi penguncian secara keseluruhan berkurang sebanyak 147.
```
